frontend/simple/views/Mailbox.vue

```diff
@@ -219,9 +200,9 @@ export default {
     },
     remove: function (index) {
       if (Number.isInteger(index)) {
-        this.$store.dispatch('deleteMessage', this.inbox[index].id)
+        this.$store.commit('deleteMessage', this.inbox[index].hash)
```

Comment:
`this.inbox` is a computed property and `index` comes from the renderer in the template